accumulo-iterators/src/main/java/org/vertexium/accumulo/iterator/util/DataOutputStreamUtils.java

```diff
@@ -26,6 +26,17 @@ public static void encodeTextList(DataOutputStream out, Collection<Text> texts)
         }
     }
 
+    public static void encodeStringSet(DataOutputStream out, Set<String> set) throws IOException {
```

Comment:
Empty sets will write out `0` and result in an empty set on the other side as opposed to `null` (-1)